06_create_cluster.sh

```diff
@@ -7,6 +7,14 @@ source utils.sh
 source common.sh
 source ocp_install_env.sh
 
+# Do some PULL_SECRET sanity checking
+if [[ "${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}" == *"registry.svc.ci.openshift.org"* ]]; then
+    if [[ "${PULL_SECRET}" != *"registry.svc.ci.openshift.org"* ]]; then
```

Comment:
This broke installation for me using the pull secret from https://cloud.openshift.com/clusters/install

The URL I have is `cloud.openshift.com` not `registry.svc.ci.openshift.org`

Reply:
You need a pull secret that has access to that registry, we have a shared one in Slack you can use on `#forum-kni-deployment.`

Reply:
You can also create one:

1) log in to https://api.ci.openshift.org/console
2) Top right, under name, 'Copy Login Command'
3) oc login https://api.ci.openshift.org --token=...
4) oc registry login --to=PULL_SECRET_FILE